doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1388,19 +1388,15 @@ Response:
 
 This operation returns the `value` property with the value of the attribute.
 
-* If the response payload MIME type is `application/json`:
-  * If the attribute value is a JSON object or array, the payload is that JSON object or array
-  * If the attribute value is a string, a number, null or a boolean, an HTTP error 
-    "406 Not Acceptable: accepted MIME types: text/plain" is returned.
-* If the response payload MIME type is `text/plain`,
-  * If the attribute value is a JSON object or array, the payload in a stringification of that JSON
-    object or array.
-  * If the attribute value is a string, number, null or boolean, the payload is a text
-    representation of that value. In case of a string, citation marks are used at the begining and
-    end.
-
-The payload MIME type in responses is based on client-server HTTP negotiation, based on the client
-`Accept` header.
+* If attribute value is JSON Array or Object:
```

Comment:
I think this can be made clearer:

If accept header does not include a MIME type which can expand to `text/plain` or `application/json` an error `406` must be raised. 

In the absence of any hint from accept headers, or if accept headers specify a generic mime type (`*/*`), an attribute value will be returned as JSON (Content-Type: `application/json`) if the attribute value is a JSON object or array. Conversely, it will be returned as plain text (Content-Type: `text/plain`) in any other case. In the particular case of a `string`, citation (`"`) marks must be used at the beginning and end.

If accept headers include only `application/json`  and the attribute value has to be represented as text plain (as per the rules above) it will be considered an error `406` Not Acceptable: accepted MIME types: text/plain. In any other case the implementation will honor the requested MIME type provided by the client.